minijinja/src/compiler/parser.rs

```diff
@@ -953,7 +954,6 @@ impl<'a> Parser<'a> {
         Ok(ast::FromImport { expr, names })
     }
 
-    #[cfg(feature = "macros")]
```

Comment:
As functions can be called via do, this flag needed to be removed, but then the follow-up commit in the other branch renamed this function. I'm not sure what to call this function now, given it also calls functions and not just macros.

Reply:
The `do` tag does not support args/defaults so I just removed that logic entirely from the parsing step.